Documentation/En/Index.xyl

```diff
@@ -114,13 +114,13 @@
   data) that fixes the values of the arguments (also called inputs) of the
   SUT. A <strong>test case</strong> is a pair composed of a state (the context
   in which the test is executed) and a test. A <strong>test suite</strong> is
-  a set of test cases. Therefore, there is two states from the test point of
+  a set of test cases. Therefore, there are two states from the test point of
   view: A <strong>pre-state</strong> —before the execution of the SUT— and a
   <strong>post-state</strong> —after the execution—.</p>
   <p>Building the pre-state is something crucial: The goal is to put the SUT into a
   specific (pre-)state in order to test it. The part of the test case
-  responsible to create the pre-state is called the <strong>preamble</strong>.
-  Thus, the preambler is a code that puts the SUT into a specific state.</p>
+  responsible for creating the pre-state is called the <strong>preamble</strong>.
```

Comment:
I am a native speaker, so I don't know much about the language theory, but, here the test case is responsible *for* something. `to` implies "direction" here, `he is responsible to the director` (he must answer to the director), is different from `he is responsible for the director` (he takes care of the director).